hw/top_earlgrey/top_earlgrey.core

```diff
@@ -40,7 +40,7 @@ filesets:
       - lowrisc:ip:usbdev
       - lowrisc:top_earlgrey:xbar_main
       - lowrisc:top_earlgrey:xbar_peri
-      - lowrisc:ip:rstmgr
+      - lowrisc:systems:rstmgr
```

Comment:
wait...you shouldnt need to do this. Inside `lowrisc:ip:rstmgr` there are the fileset redirect flags.

Reply:
can you let me know the exact error / warning you were seeing?

Reply:
oh I see. let me revert that. I still wonder why ascentlint is unhappy about this on the top-level, though...